Makefile

```diff
@@ -128,7 +128,11 @@ deploy-s3:
 	# deploy to hashed urls on s3
 	nht deploy-hashed-assets --directory public/n-ui --monitor-assets
 
-deploy: deploy-s3 npm-publish rebuild-user-facing-apps
+publish-npm:
```

Comment:
can this just go into n-gage? maybe make the npm-publish task run this code if it detects the installed npm supports these options, otherwise fallback to the old version